connect_your_app_to_IRIS/technical_details/app_onboarding.md

```diff
@@ -94,15 +216,66 @@ Sending messages to IRIS is done with JSON-RPC.
     
 You can find the specification of the respective methods and destinations [here](#destinations-and-methods).
 
-## Destinations and methods
+To test, we will simply send a test location to IRIS staging:
 
-### Location search index
+    curl --header "Content-Type: application/json" --request POST --insecure --data '
+    {
+        "method":"ls-1.postLocationsToSearchIndex",
+        "id": "1",
+        "params": {
+            "locations": [
+                {
+                    "id": "1abc",
+                    "name": "Visits Test"
+                }
+            ]
+        },
+        "jsonrpc": "2.0"
+    }
+    ' https://localhost:5556/jsonrpc
 
-Location search index can be reached at `ls-1`.
+We should get 
+
+    {
+        "jsonrpc" : "2.0",
+        "result" : "OK",
+        "id" : "1"
+    }
+
+It is important to know, what is going wrong if you won't get this response. If you post your request to a wrong endpoint, you would just get a "message":"not found". If the port 5556 is not bound to the eps container, you will get a "connection refused".
+
+| cURL result | Possible cause | Ckecks to perform |
+|-|-| - |
+| Client sent an HTTP request to an HTTPS server. | You called the endpoint at http:// instead of https:// | Ckeck the address in cURL command and options you used. |
+| Connection refused | The EPS is not listening on the URI or port of the request. | Ckeck the address in cURL command. Are you running the command from localhost - the docker host, or do you need to address an IP address in your network? Is there a firewall blocking the port? Is the EPS running - check it with docker ps, and check the logs. |
+| "message":"not found" | You addressed wrong endpoint | If you address https://localhost:5556/ (without endpoint address jsonrpc) you will get this result. |
+| {"jsonrpc":"2.0","error":{"code":-32603,"message":"internal error"},"id":"1"} | There is a bug in EPS or your configuration is incorrect. | Check the settings yml, try another EPS version. |
```

Comment:
I welcome the fact that we are dealing with the possible errors. We can go into more detail here at a later date.